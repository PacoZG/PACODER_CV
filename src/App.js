import Header from './components/header'
import Body from './components/Body'
import Footer from './components/Footer'

const App = () => {
  return (
    <div className=" w-screen">
      <Header />
      <Body />
      <Footer />
    </div>
  )
}

export default App
