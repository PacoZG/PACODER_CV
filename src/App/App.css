.App {
  background-image: url("https://res.cloudinary.com/dbn5gpgi5/image/upload/c_scale,h_720,w_140/v1640678894/fcecx1i2xc2d9vumxyjj.jpg");
  background-repeat: no-repeat;
  background-size: cover;
}

@media (min-width: 768px) {
  .App {
    background-image: url("https://res.cloudinary.com/dbn5gpgi5/image/upload/c_scale,h_1560,w_768/v1640678894/fcecx1i2xc2d9vumxyjj.jpg");
    background-repeat: no-repeat;
    background-size: cover;
  }
}

@media (min-width: 1040px) {
  .App {
    background-image: url("https://res.cloudinary.com/dbn5gpgi5/image/upload/c_scale,h_900,w_980/v1640678894/fcecx1i2xc2d9vumxyjj.jpg");
    background-repeat: no-repeat;
    background-size: cover;
  }
}

@media (min-width: 2560px) {
  .App {
    background-image: url("https://res.cloudinary.com/dbn5gpgi5/image/upload/c_scale,h_1280,w_1920/v1640678894/fcecx1i2xc2d9vumxyjj.jpg");
    background-repeat: no-repeat;
    background-size: cover;
  }
}