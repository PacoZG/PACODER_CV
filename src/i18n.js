import i18n from 'i18next'
import { initReactI18next } from 'react-i18next'
import LanguageDetector from 'i18next-browser-languagedetector'

import translationEN from './utils/language/en/translations.json'
import translationES from './utils/language/es/translations.json'

i18n
  .use(LanguageDetector)
  .use(initReactI18next)
  .init({
    debug: true,
    fallbackLng: 'EN',
    interpolation: {
      escapeValue: false, // not needed for react as it escapes by default
    },
    resources: {
      EN: {
        translation: translationEN,
      },
      ES: {
        translation: translationES,
      },
    },
  })

export default i18n
