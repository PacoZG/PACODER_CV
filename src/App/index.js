import React from 'react'
import { BrowserRouter, Navigate, Route, Routes } from 'react-router-dom'
import Contact from '../components/Contact'
import Experience from '../components/Experience'
import Footer from '../components/Footer'
import Header from '../components/Header'
import Presentation from '../components/Presentation'
import Div from '../components/private/Div'
import WhatYouGet from '../components/WhatYouGet'
import './App.css'
// import Body from '../components/Body'

const App = () => {
  return (
    <Div className="App">
      <BrowserRouter>
        <Header />
        <Routes>
          <Route path="/" element={<Navigate to="/cv/presentation" />} />
          <Route path="/cv/presentation" element={<Presentation />} />
          <Route path="/cv/skills_strenghts" element={<WhatYouGet />} />
          <Route path="/cv/experience" element={<Experience />} />
          <Route path={'/cv/contact'} element={<Contact />} />
          {/* <Route path="/forum" element={<Body />} /> */}
        </Routes>
        <Footer />
      </BrowserRouter>
    </Div>
  )
}

export default App
