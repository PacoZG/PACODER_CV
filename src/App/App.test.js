import React from 'react'
import renderer from 'react-test-renderer'
import { BrowserRouter } from 'react-router-dom'
import { I18nextProvider } from 'react-i18next'
import i18n from '../utils/language/i18nextForTest'
import App from '.'

describe('Testing header', () => {
  test('Snapshot of the Main App component', () => {
    const rederedHeader = renderer
      .create(
        <I18nextProvider i18n={i18n}>
          <App />
        </I18nextProvider>
      )
      .toJSON()
    expect(rederedHeader).toMatchSnapshot()
  })
})
