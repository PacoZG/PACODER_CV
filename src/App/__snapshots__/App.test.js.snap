// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing header Snapshot of the Main App component 1`] = `
<div
  className="App"
>
  <div
    className="fixed z-10 w-screen bg-transparent transition duration-500"
  >
    <div
      className="header-style relative lg:hidden flex flex-row items-center justify-center mx-3 m-2"
    >
      <div
        className="flex justify-center w-full"
      >
        <div>
          <div
            className="block md:hidden"
          >
            <div
              className="flex justify-end items-center space-x-2"
            >
              <div>
                <input
                  className="hidden"
                  id="toggle"
                  name=""
                  type="checkbox"
                />
                <span
                  className="relative w-14 h-7 flex items-center shadow-lg bg-gray-300 dark:bg-gray-300 rounded-full p-1 cursor-pointer"
                  onClick={[Function]}
                >
                  <svg
                    aria-hidden="true"
                    className="absolute left-1 h-6 w-6 transition duration-500 text-gray-800 text-opacity-0 dark:text-opacity-100"
                    fill="currentColor"
                    viewBox="0 0 20 20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clipRule="evenodd"
                      d="M10 2a1 1 0 011 1v1a1 1 0 11-2 0V3a1 1 0 011-1zm4 8a4 4 0 11-8 0 4 4 0 018 0zm-.464 4.95l.707.707a1 1 0 001.414-1.414l-.707-.707a1 1 0 00-1.414 1.414zm2.12-10.607a1 1 0 010 1.414l-.706.707a1 1 0 11-1.414-1.414l.707-.707a1 1 0 011.414 0zM17 11a1 1 0 100-2h-1a1 1 0 100 2h1zm-7 4a1 1 0 011 1v1a1 1 0 11-2 0v-1a1 1 0 011-1zM5.05 6.464A1 1 0 106.465 5.05l-.708-.707a1 1 0 00-1.414 1.414l.707.707zm1.414 8.486l-.707.707a1 1 0 01-1.414-1.414l.707-.707a1 1 0 011.414 1.414zM4 11a1 1 0 100-2H3a1 1 0 000 2h1z"
                      fillRule="evenodd"
                    />
                  </svg>
                  <div
                    className="w-6 h-6 bg-blue-800 rounded-full shadow-md transition duration-500"
                  />
                  <svg
                    aria-hidden="true"
                    className="absolute right-1 h-5 w-5 transition duration-500 text-gray-800 text-opacity-100 dark:text-opacity-0"
                    fill="currentColor"
                    viewBox="0 0 20 20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M17.293 13.293A8 8 0 016.707 2.707a8.001 8.001 0 1010.586 10.586z"
                    />
                  </svg>
                </span>
              </div>
            </div>
          </div>
          <div
            className="hidden md:block"
          >
            <div
              className="flex justify-end items-center space-x-2"
            >
              <svg
                aria-hidden="true"
                className="h-6 w-6 transition duration-500 dark:text-gray-300"
                fill="currentColor"
                viewBox="0 0 20 20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clipRule="evenodd"
                  d="M10 2a1 1 0 011 1v1a1 1 0 11-2 0V3a1 1 0 011-1zm4 8a4 4 0 11-8 0 4 4 0 018 0zm-.464 4.95l.707.707a1 1 0 001.414-1.414l-.707-.707a1 1 0 00-1.414 1.414zm2.12-10.607a1 1 0 010 1.414l-.706.707a1 1 0 11-1.414-1.414l.707-.707a1 1 0 011.414 0zM17 11a1 1 0 100-2h-1a1 1 0 100 2h1zm-7 4a1 1 0 011 1v1a1 1 0 11-2 0v-1a1 1 0 011-1zM5.05 6.464A1 1 0 106.465 5.05l-.708-.707a1 1 0 00-1.414 1.414l.707.707zm1.414 8.486l-.707.707a1 1 0 01-1.414-1.414l.707-.707a1 1 0 011.414 1.414zM4 11a1 1 0 100-2H3a1 1 0 000 2h1z"
                  fillRule="evenodd"
                />
              </svg>
              <div>
                <input
                  className="hidden"
                  id="toggle"
                  name=""
                  type="checkbox"
                />
                <span
                  className="w-9 h-5 flex items-center shadow-lg bg-gray-300 dark:bg-gray-300 rounded-full p-1 cursor-pointer"
                  onClick={[Function]}
                >
                  <div
                    className="w-4 h-4 bg-blue-800 rounded-full shadow-md transition duration-100"
                  />
                </span>
              </div>
              <svg
                aria-hidden="true"
                className="h-5 w-5 transition duration-500 dark:text-gray-300 "
                fill="currentColor"
                viewBox="0 0 20 20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M17.293 13.293A8 8 0 016.707 2.707a8.001 8.001 0 1010.586 10.586z"
                />
              </svg>
            </div>
          </div>
        </div>
      </div>
      <div
        className="flex items-center space-x-2 md:space-x-6 p-2"
      >
        <a
          href="https://www.linkedin.com/in/francisco-zavala/"
          target="blank"
        >
          <svg
            height={31}
            viewBox="0 0 64 64"
            width={31}
          >
            <rect
              fill="#007fb1"
              height="64"
              rx={10}
              ry={10}
              style={Object {}}
              width="64"
            />
            <path
              d="M20.4,44h5.4V26.6h-5.4V44z M23.1,18c-1.7,0-3.1,1.4-3.1,3.1c0,1.7,1.4,3.1,3.1,3.1 c1.7,0,3.1-1.4,3.1-3.1C26.2,19.4,24.8,18,23.1,18z M39.5,26.2c-2.6,0-4.4,1.4-5.1,2.8h-0.1v-2.4h-5.2V44h5.4v-8.6 c0-2.3,0.4-4.5,3.2-4.5c2.8,0,2.8,2.6,2.8,4.6V44H46v-9.5C46,29.8,45,26.2,39.5,26.2z"
              fill="white"
            />
          </svg>
        </a>
        <a
          href="https://github.com/PacoZG"
          target="blank"
        >
          <svg
            className="bg-gray-300 h-8 w-8 rounded-md"
          >
            github.svg
          </svg>
        </a>
        <a
          href="https://wa.me/358449888032?text=I%20will%20like%20to%20contact%20you%20to%20arrange%20an%20interview"
          target="blank"
        >
          <svg
            height={31}
            viewBox="0 0 64 64"
            width={31}
          >
            <rect
              fill="#25D366"
              height="64"
              rx={10}
              ry={10}
              style={Object {}}
              width="64"
            />
            <path
              d="m42.32286,33.93287c-0.5178,-0.2589 -3.04726,-1.49644 -3.52105,-1.66732c-0.4712,-0.17346 -0.81554,-0.2589 -1.15987,0.2589c-0.34175,0.51004 -1.33075,1.66474 -1.63108,2.00648c-0.30032,0.33658 -0.60064,0.36247 -1.11327,0.12945c-0.5178,-0.2589 -2.17994,-0.80259 -4.14759,-2.56312c-1.53269,-1.37217 -2.56312,-3.05503 -2.86603,-3.57283c-0.30033,-0.5178 -0.03366,-0.80259 0.22524,-1.06149c0.23301,-0.23301 0.5178,-0.59547 0.7767,-0.90616c0.25372,-0.31068 0.33657,-0.5178 0.51262,-0.85437c0.17088,-0.36246 0.08544,-0.64725 -0.04402,-0.90615c-0.12945,-0.2589 -1.15987,-2.79613 -1.58964,-3.80584c-0.41424,-1.00971 -0.84142,-0.88027 -1.15987,-0.88027c-0.29773,-0.02588 -0.64208,-0.02588 -0.98382,-0.02588c-0.34693,0 -0.90616,0.12945 -1.37736,0.62136c-0.4712,0.5178 -1.80194,1.76053 -1.80194,4.27186c0,2.51134 1.84596,4.945 2.10227,5.30747c0.2589,0.33657 3.63497,5.51458 8.80262,7.74113c1.23237,0.5178 2.1903,0.82848 2.94111,1.08738c1.23237,0.38836 2.35599,0.33657 3.24402,0.20712c0.99159,-0.15534 3.04985,-1.24272 3.47963,-2.45956c0.44013,-1.21683 0.44013,-2.22654 0.31068,-2.45955c-0.12945,-0.23301 -0.46601,-0.36247 -0.98382,-0.59548m-9.40068,12.84407l-0.02589,0c-3.05503,0 -6.08417,-0.82849 -8.72495,-2.38189l-0.62136,-0.37023l-6.47252,1.68286l1.73463,-6.29129l-0.41424,-0.64725c-1.70875,-2.71846 -2.6149,-5.85116 -2.6149,-9.07706c0,-9.39809 7.68934,-17.06155 17.15993,-17.06155c4.58253,0 8.88029,1.78642 12.11655,5.02268c3.23625,3.21036 5.02267,7.50812 5.02267,12.06476c-0.0078,9.3981 -7.69712,17.06155 -17.14699,17.06155m14.58906,-31.58846c-3.93529,-3.80584 -9.1133,-5.95471 -14.62789,-5.95471c-11.36055,0 -20.60848,9.2065 -20.61625,20.52564c0,3.61684 0.94757,7.14565 2.75211,10.26282l-2.92557,10.63564l10.93337,-2.85309c3.0136,1.63108 6.4052,2.4958 9.85634,2.49839l0.01037,0c11.36574,0 20.61884,-9.2091 20.62403,-20.53082c0,-5.48093 -2.14111,-10.64081 -6.03239,-14.51915"
              fill="white"
            />
          </svg>
        </a>
        <a
          href="/cv/contact"
          onClick={[Function]}
        >
          <svg
            height={31}
            viewBox="0 0 64 64"
            width={31}
          >
            <rect
              fill="#7f7f7f"
              height="64"
              rx={10}
              ry={10}
              style={Object {}}
              width="64"
            />
            <path
              d="M17,22v20h30V22H17z M41.1,25L32,32.1L22.9,25H41.1z M20,39V26.6l12,9.3l12-9.3V39H20z"
              fill="white"
            />
          </svg>
        </a>
      </div>
      <div
        className="flex justify-center w-full"
      >
        <button
          className="mobile-lang-text-style"
          onClick={[Function]}
        >
          ESP
        </button>
      </div>
      <div
        className="header-style link-menu"
      >
        <div
          className="flex items-center h-12 w-9 "
        >
          <svg
            aria-hidden="true"
            className=" text-gray-200 border-r border-gray-300 p-1 h-6 w-6"
            fill="currentColor"
            viewBox="0 0 20 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clipRule="evenodd"
              d="M3 5a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 10a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 15a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1z"
              fillRule="evenodd"
            />
          </svg>
        </div>
        <div
          className="mobile-link-group"
        >
          <a
            className="mobile-link"
            href="/cv/presentation"
            onClick={[Function]}
          >
            Presentation
          </a>
          <a
            className="mobile-link"
            href="/cv/skills_strenghts"
            onClick={[Function]}
          >
            Skills & Strenghts
          </a>
          <a
            className="mobile-link"
            href="/cv/experience"
            onClick={[Function]}
          >
            Working Experience
          </a>
          <a
            className="mobile-link"
            href="/cv/contact"
            onClick={[Function]}
          >
            Contact
          </a>
        </div>
      </div>
    </div>
    <div
      className="hidden lg:block"
    >
      <div
        className="header-style flex flex-row items-center justify-evenly p-1 shadow-2xl  mr-14 ml-10 m-2"
      >
        <div>
          <div
            className="block md:hidden"
          >
            <div
              className="flex justify-end items-center space-x-2"
            >
              <div>
                <input
                  className="hidden"
                  id="toggle"
                  name=""
                  type="checkbox"
                />
                <span
                  className="relative w-14 h-7 flex items-center shadow-lg bg-gray-300 dark:bg-gray-300 rounded-full p-1 cursor-pointer"
                  onClick={[Function]}
                >
                  <svg
                    aria-hidden="true"
                    className="absolute left-1 h-6 w-6 transition duration-500 text-gray-800 text-opacity-0 dark:text-opacity-100"
                    fill="currentColor"
                    viewBox="0 0 20 20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clipRule="evenodd"
                      d="M10 2a1 1 0 011 1v1a1 1 0 11-2 0V3a1 1 0 011-1zm4 8a4 4 0 11-8 0 4 4 0 018 0zm-.464 4.95l.707.707a1 1 0 001.414-1.414l-.707-.707a1 1 0 00-1.414 1.414zm2.12-10.607a1 1 0 010 1.414l-.706.707a1 1 0 11-1.414-1.414l.707-.707a1 1 0 011.414 0zM17 11a1 1 0 100-2h-1a1 1 0 100 2h1zm-7 4a1 1 0 011 1v1a1 1 0 11-2 0v-1a1 1 0 011-1zM5.05 6.464A1 1 0 106.465 5.05l-.708-.707a1 1 0 00-1.414 1.414l.707.707zm1.414 8.486l-.707.707a1 1 0 01-1.414-1.414l.707-.707a1 1 0 011.414 1.414zM4 11a1 1 0 100-2H3a1 1 0 000 2h1z"
                      fillRule="evenodd"
                    />
                  </svg>
                  <div
                    className="w-6 h-6 bg-blue-800 rounded-full shadow-md transition duration-500"
                  />
                  <svg
                    aria-hidden="true"
                    className="absolute right-1 h-5 w-5 transition duration-500 text-gray-800 text-opacity-100 dark:text-opacity-0"
                    fill="currentColor"
                    viewBox="0 0 20 20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M17.293 13.293A8 8 0 016.707 2.707a8.001 8.001 0 1010.586 10.586z"
                    />
                  </svg>
                </span>
              </div>
            </div>
          </div>
          <div
            className="hidden md:block"
          >
            <div
              className="flex justify-end items-center space-x-2"
            >
              <svg
                aria-hidden="true"
                className="h-6 w-6 transition duration-500 dark:text-gray-300"
                fill="currentColor"
                viewBox="0 0 20 20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clipRule="evenodd"
                  d="M10 2a1 1 0 011 1v1a1 1 0 11-2 0V3a1 1 0 011-1zm4 8a4 4 0 11-8 0 4 4 0 018 0zm-.464 4.95l.707.707a1 1 0 001.414-1.414l-.707-.707a1 1 0 00-1.414 1.414zm2.12-10.607a1 1 0 010 1.414l-.706.707a1 1 0 11-1.414-1.414l.707-.707a1 1 0 011.414 0zM17 11a1 1 0 100-2h-1a1 1 0 100 2h1zm-7 4a1 1 0 011 1v1a1 1 0 11-2 0v-1a1 1 0 011-1zM5.05 6.464A1 1 0 106.465 5.05l-.708-.707a1 1 0 00-1.414 1.414l.707.707zm1.414 8.486l-.707.707a1 1 0 01-1.414-1.414l.707-.707a1 1 0 011.414 1.414zM4 11a1 1 0 100-2H3a1 1 0 000 2h1z"
                  fillRule="evenodd"
                />
              </svg>
              <div>
                <input
                  className="hidden"
                  id="toggle"
                  name=""
                  type="checkbox"
                />
                <span
                  className="w-9 h-5 flex items-center shadow-lg bg-gray-300 dark:bg-gray-300 rounded-full p-1 cursor-pointer"
                  onClick={[Function]}
                >
                  <div
                    className="w-4 h-4 bg-blue-800 rounded-full shadow-md transition duration-100"
                  />
                </span>
              </div>
              <svg
                aria-hidden="true"
                className="h-5 w-5 transition duration-500 dark:text-gray-300 "
                fill="currentColor"
                viewBox="0 0 20 20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M17.293 13.293A8 8 0 016.707 2.707a8.001 8.001 0 1010.586 10.586z"
                />
              </svg>
            </div>
          </div>
        </div>
        <div
          className="flex items-center space-x-1 md:space-x-6 p-2"
        >
          <a
            href="https://www.linkedin.com/in/francisco-zavala/"
            target="blank"
          >
            <svg
              height={46}
              viewBox="0 0 64 64"
              width={46}
            >
              <rect
                fill="#007fb1"
                height="64"
                rx={10}
                ry={10}
                style={Object {}}
                width="64"
              />
              <path
                d="M20.4,44h5.4V26.6h-5.4V44z M23.1,18c-1.7,0-3.1,1.4-3.1,3.1c0,1.7,1.4,3.1,3.1,3.1 c1.7,0,3.1-1.4,3.1-3.1C26.2,19.4,24.8,18,23.1,18z M39.5,26.2c-2.6,0-4.4,1.4-5.1,2.8h-0.1v-2.4h-5.2V44h5.4v-8.6 c0-2.3,0.4-4.5,3.2-4.5c2.8,0,2.8,2.6,2.8,4.6V44H46v-9.5C46,29.8,45,26.2,39.5,26.2z"
                fill="white"
              />
            </svg>
          </a>
          <a
            href="https://github.com/PacoZG"
          >
            <svg
              className="bg-gray-300  h-11 w-11 rounded-md"
            >
              github.svg
            </svg>
          </a>
          <a
            href="https://wa.me/358449888032?text=I%20will%20like%20to%20contact%20you%20to%20arrange%20an%20interview"
            target="blank"
          >
            <svg
              height={46}
              viewBox="0 0 64 64"
              width={46}
            >
              <rect
                fill="#25D366"
                height="64"
                rx={10}
                ry={10}
                style={Object {}}
                width="64"
              />
              <path
                d="m42.32286,33.93287c-0.5178,-0.2589 -3.04726,-1.49644 -3.52105,-1.66732c-0.4712,-0.17346 -0.81554,-0.2589 -1.15987,0.2589c-0.34175,0.51004 -1.33075,1.66474 -1.63108,2.00648c-0.30032,0.33658 -0.60064,0.36247 -1.11327,0.12945c-0.5178,-0.2589 -2.17994,-0.80259 -4.14759,-2.56312c-1.53269,-1.37217 -2.56312,-3.05503 -2.86603,-3.57283c-0.30033,-0.5178 -0.03366,-0.80259 0.22524,-1.06149c0.23301,-0.23301 0.5178,-0.59547 0.7767,-0.90616c0.25372,-0.31068 0.33657,-0.5178 0.51262,-0.85437c0.17088,-0.36246 0.08544,-0.64725 -0.04402,-0.90615c-0.12945,-0.2589 -1.15987,-2.79613 -1.58964,-3.80584c-0.41424,-1.00971 -0.84142,-0.88027 -1.15987,-0.88027c-0.29773,-0.02588 -0.64208,-0.02588 -0.98382,-0.02588c-0.34693,0 -0.90616,0.12945 -1.37736,0.62136c-0.4712,0.5178 -1.80194,1.76053 -1.80194,4.27186c0,2.51134 1.84596,4.945 2.10227,5.30747c0.2589,0.33657 3.63497,5.51458 8.80262,7.74113c1.23237,0.5178 2.1903,0.82848 2.94111,1.08738c1.23237,0.38836 2.35599,0.33657 3.24402,0.20712c0.99159,-0.15534 3.04985,-1.24272 3.47963,-2.45956c0.44013,-1.21683 0.44013,-2.22654 0.31068,-2.45955c-0.12945,-0.23301 -0.46601,-0.36247 -0.98382,-0.59548m-9.40068,12.84407l-0.02589,0c-3.05503,0 -6.08417,-0.82849 -8.72495,-2.38189l-0.62136,-0.37023l-6.47252,1.68286l1.73463,-6.29129l-0.41424,-0.64725c-1.70875,-2.71846 -2.6149,-5.85116 -2.6149,-9.07706c0,-9.39809 7.68934,-17.06155 17.15993,-17.06155c4.58253,0 8.88029,1.78642 12.11655,5.02268c3.23625,3.21036 5.02267,7.50812 5.02267,12.06476c-0.0078,9.3981 -7.69712,17.06155 -17.14699,17.06155m14.58906,-31.58846c-3.93529,-3.80584 -9.1133,-5.95471 -14.62789,-5.95471c-11.36055,0 -20.60848,9.2065 -20.61625,20.52564c0,3.61684 0.94757,7.14565 2.75211,10.26282l-2.92557,10.63564l10.93337,-2.85309c3.0136,1.63108 6.4052,2.4958 9.85634,2.49839l0.01037,0c11.36574,0 20.61884,-9.2091 20.62403,-20.53082c0,-5.48093 -2.14111,-10.64081 -6.03239,-14.51915"
                fill="white"
              />
            </svg>
          </a>
          <a
            href="/cv/contact"
            onClick={[Function]}
          >
            <svg
              height={46}
              viewBox="0 0 64 64"
              width={46}
            >
              <rect
                fill="#7f7f7f"
                height="64"
                rx={10}
                ry={10}
                style={Object {}}
                width="64"
              />
              <path
                d="M17,22v20h30V22H17z M41.1,25L32,32.1L22.9,25H41.1z M20,39V26.6l12,9.3l12-9.3V39H20z"
                fill="white"
              />
            </svg>
          </a>
        </div>
        <button
          className="pc-lang-text-style"
          onClick={[Function]}
        >
          ESP
        </button>
        <div
          className="header-style web-link-menu"
        >
          <div
            className="icon-container"
          >
            <svg
              aria-hidden="true"
              className="menu-icon"
              fill="currentColor"
              viewBox="0 0 20 20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clipRule="evenodd"
                d="M3 5a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 10a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 15a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1z"
                fillRule="evenodd"
              />
            </svg>
            <svg
              aria-hidden="true"
              className="x-icon"
              fill="currentColor"
              viewBox="0 0 20 20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clipRule="evenodd"
                d="M4.293 4.293a1 1 0 011.414 0L10 8.586l4.293-4.293a1 1 0 111.414 1.414L11.414 10l4.293 4.293a1 1 0 01-1.414 1.414L10 11.414l-4.293 4.293a1 1 0 01-1.414-1.414L8.586 10 4.293 5.707a1 1 0 010-1.414z"
                fillRule="evenodd"
              />
            </svg>
          </div>
          <div
            className="pc-link-group"
          >
            <a
              className="link"
              href="/cv/presentation"
              onClick={[Function]}
            >
              Presentation
            </a>
            <a
              className="link"
              href="/cv/skills_strenghts"
              onClick={[Function]}
            >
              Skills & Strenghts
            </a>
            <a
              className="link"
              href="/cv/experience"
              onClick={[Function]}
            >
              Working Experience
            </a>
            <a
              className="link"
              href="/cv/contact"
              onClick={[Function]}
            >
              Contact
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="relative inset-x-0 bottom-0 p-4 pt-3 pb-3 bg-blue-1000 border-t border-black dark:border-gray-200 transition duration-500"
  >
    <div
      className="flex flex-col md:flex-row justify-around items-center md:items-start space-y-2 md:space-y-0 md:p-2"
    >
      <div
        className="flex flex-col items-center md:items-start space-y-2"
      >
        <div
          className="flex flex-col items-center md:items-start space-y-2 md:space-y-0 "
        >
          <div
            className="flex items-center space-x-1"
          >
            <svg
              aria-hidden="true"
              className="h-3 w-3 text-gray-300"
              fill="currentColor"
              viewBox="0 0 20 20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clipRule="evenodd"
                d="M5.05 4.05a7 7 0 119.9 9.9L10 18.9l-4.95-4.95a7 7 0 010-9.9zM10 11a2 2 0 100-4 2 2 0 000 4z"
                fillRule="evenodd"
              />
            </svg>
            <p
              className="text-gray-300 text-sm md:pl-1"
            >
              Helsinki, Finland
            </p>
          </div>
        </div>
        <div
          className="flex space-x-1 items-center"
        >
          <svg
            aria-hidden="true"
            className="h-4 w-4 text-gray-300"
            fill="currentColor"
            viewBox="0 0 20 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clipRule="evenodd"
              d="M14.243 5.757a6 6 0 10-.986 9.284 1 1 0 111.087 1.678A8 8 0 1118 10a3 3 0 01-4.8 2.401A4 4 0 1114 10a1 1 0 102 0c0-1.537-.586-3.07-1.757-4.243zM12 10a2 2 0 10-4 0 2 2 0 004 0z"
              fillRule="evenodd"
            />
          </svg>
          <a
            className="text-gray-300 text-sm space-y-2 md:space-y-0 "
            href="/cv/contact"
            onClick={[Function]}
          >
            contact@pacoderzavala.com
          </a>
        </div>
      </div>
      <div>
        <div
          className="flex flex-col items-center space-y-2"
        >
          <p
            className="text-center text-gray-300 text-xs font-semibold"
          >
            SHARE MY WEBSITE
          </p>
          <div
            className="flex items-center justify-between gap-4"
          >
            <div>
              <button
                aria-label="facebook"
                className="react-share__ShareButton  font-medium focus-within:outline-none "
                onClick={[Function]}
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "border": "none",
                    "color": "inherit",
                    "cursor": "pointer",
                    "font": "inherit",
                    "padding": 0,
                  }
                }
              >
                <svg
                  height={24}
                  viewBox="0 0 64 64"
                  width={24}
                >
                  <circle
                    cx="32"
                    cy="32"
                    fill="#3b5998"
                    r="31"
                    style={Object {}}
                  />
                  <path
                    d="M34.1,47V33.3h4.6l0.7-5.3h-5.3v-3.4c0-1.5,0.4-2.6,2.6-2.6l2.8,0v-4.8c-0.5-0.1-2.2-0.2-4.1-0.2 c-4.1,0-6.9,2.5-6.9,7V28H24v5.3h4.6V47H34.1z"
                    fill="white"
                  />
                </svg>
              </button>
            </div>
            <div>
              <button
                aria-label="linkedin"
                className="react-share__ShareButton  font-medium focus-within:outline-none "
                onClick={[Function]}
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "border": "none",
                    "color": "inherit",
                    "cursor": "pointer",
                    "font": "inherit",
                    "padding": 0,
                  }
                }
              >
                <svg
                  height={24}
                  viewBox="0 0 64 64"
                  width={24}
                >
                  <circle
                    cx="32"
                    cy="32"
                    fill="#007fb1"
                    r="31"
                    style={Object {}}
                  />
                  <path
                    d="M20.4,44h5.4V26.6h-5.4V44z M23.1,18c-1.7,0-3.1,1.4-3.1,3.1c0,1.7,1.4,3.1,3.1,3.1 c1.7,0,3.1-1.4,3.1-3.1C26.2,19.4,24.8,18,23.1,18z M39.5,26.2c-2.6,0-4.4,1.4-5.1,2.8h-0.1v-2.4h-5.2V44h5.4v-8.6 c0-2.3,0.4-4.5,3.2-4.5c2.8,0,2.8,2.6,2.8,4.6V44H46v-9.5C46,29.8,45,26.2,39.5,26.2z"
                    fill="white"
                  />
                </svg>
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        className="flex flex-col items-center md:items-start space-y-1"
      >
        <p
          className="text-sm text-gray-300"
        >
          Developed by:
        </p>
        <a
          className="hover:text-gray-400 text-gray-300 text-sm"
          href="https://www.linkedin.com/in/francisco-zavala"
          target="blank"
        >
          <div
            className="flex items-center space-x-1"
          >
            <svg
              height={28}
              viewBox="0 0 64 64"
              width={28}
            >
              <circle
                cx="32"
                cy="32"
                fill="#007fb1"
                r="31"
                style={Object {}}
              />
              <path
                d="M20.4,44h5.4V26.6h-5.4V44z M23.1,18c-1.7,0-3.1,1.4-3.1,3.1c0,1.7,1.4,3.1,3.1,3.1 c1.7,0,3.1-1.4,3.1-3.1C26.2,19.4,24.8,18,23.1,18z M39.5,26.2c-2.6,0-4.4,1.4-5.1,2.8h-0.1v-2.4h-5.2V44h5.4v-8.6 c0-2.3,0.4-4.5,3.2-4.5c2.8,0,2.8,2.6,2.8,4.6V44H46v-9.5C46,29.8,45,26.2,39.5,26.2z"
                fill="white"
              />
            </svg>
            <p>
              Francisco Zavala
            </p>
          </div>
        </a>
      </div>
    </div>
  </div>
</div>
`;
